### ISSUE TITLE
Script error in MacOS Catalina

### ISSUE THREAD

**Issue by pinakighatak** [jldeen/dotfiles#21]
Running the bash curl on MacOS Catalina results in this error:
bash: line 3: ./script/prompt: No such file or directory

**Comment by jldeen** [comment]
Confirmed. Replicated on macOS Catalina 10.15.5 (Beta). Looking into this and will update ASAP.

**Comment by aaroncoville** [comment]
I recently forked this code and am working through a few gotchas, this one being the first.

The source is relative to execution, so one workaround for you @pinakighatak would be to clone the repo and then run the remote configure.sh from that directory. You obviously could just run the configure.sh in the repo as well. In either case, it will bring in ./script/prompt from your local system.

@jldeen I solved this in my fork by changing the source command for ./script/prompt to

`source /dev/stdin <<< "$(curl -fsSL https://raw.githubusercontent.com/jldeen/dotfiles/mac/script/prompt)"`

I also had to change the execution call to 

`bash <(curl -fsSL https://raw.githubusercontent.com/jldeen/dotfiles/mac/configure.sh)`
